.coderabbit.yaml

```
@@ -0,0 +1,70 @@
+  sequence_diagrams: true
+  assess_linked_issues: true
+  related_issues: true
+  related_prs: true
+  suggested_labels: true
+  suggested_reviewers: true
+  poem: true
+  
+  # Path filters and instructions
+  path_filters: []  # Add glob patterns to include/exclude files
+  path_instructions: []  # Add path-specific review instructions
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)